ext/curl/curl.stub.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+/** @param resource $handle */
+function curl_close($handle): void {}
+
+/**
+ * @param resource $handle
+ *
+ * @return resource|false
+ */
+function curl_copy_handle($handle) {}
+
+/** @param resource $handle */
+function curl_errno($handle): int {}
+
+/** @param resource $handle */
+function curl_error($handle): string {}
+
+/**
+ * @param resource $handle
+ *
+ * @return string|false
+ */
+function curl_escape($handle, string $string) {}
+
+/**
+ * @param resource $handle
+ *
+ * @return mixed|false
+ */
+function curl_exec($handle) {}
+
+/** @return CURLFile */
+function curl_file_create(
+    string $filename,
+    string $mimetype = UNKNOWN,
+    string $postname = UNKNOWN
+) {}
+
+function curl_file_name(string $name): void {}
```

Comment:
This function doesn't seem to exist?

Reply:
tru, removing but what about

`PHP_ME(CURLFile,			setMimeType,        arginfo_curlfile_name, ZEND_ACC_PUBLIC)`

that arginfo doesn't exist

Reply:
ah https://github.com/php/php-src/pull/4557/files#r315212410